src/plugins/lua/mx_check.lua

```diff
@@ -305,6 +311,14 @@ if opts then
     one_shot = true,
     one_param = true,
   })
+  rspamd_config:set_metric_symbol({
+    name = settings.symbol_white_mx,
+    score = -0.00,
+    description = 'Domain has whitelist MX',
```

Comment:
That's not true. We are whitelisting the domain itself, not MX.
What about 'Domain is whitelisted from MX check' ?

Reply:
Ok, I hurried )